operations/validate.go

```diff
@@ -77,25 +77,15 @@ func validateFile(path string, ac *legacyClient, quiet bool) error {
 	}
 
 	if quiet { // only return errors
-		allErrors := projErrors
-		projErrors = []validator.ValidationError{}
-		for _, e := range allErrors {
-			if e.Level == validator.Error {
-				projErrors = append(projErrors, e)
-			}
-		}
+		projErrors = projErrors.AtLevel(validator.Error)
 	}
-	if len(projErrors) == 0 {
-		grip.Infof("%s is valid", path)
-	} else {
+
+	if len(projErrors) != 0 {
 		grip.Info(projErrors)
```

Comment:
This log line seems unnecessary

Reply:
What do you mean? If the project contains errors, this prints the validation errors.